packages/cicero-cli/test/cli.js

```diff
@@ -1191,21 +1211,24 @@ describe('#archive', async () => {
     it('should create an Ergo archive', async () => {
         const tmpFile = await tmp.file();
         const tmpArchive = tmpFile.path + '.cta';
-        await Commands.archive(template, 'ergo', tmpArchive, false);
+        const options = {};
+        await Commands.archive(template, 'ergo', tmpArchive, options);
```

Comment:
Should this be, `Commands.archive(template, 'ergo', tmpArchive, false, options);`?

Reply:
Since the signing functionality is [optional](https://github.com/sanketshevkar/cicero/blob/93c864d6103891694dbe2eec61caac2388c71ad8/packages/cicero-cli/index.js#L481) in toArchive method, before signing the template, it checks if inside [options object, keystore](https://github.com/sanketshevkar/cicero/blob/93c864d6103891694dbe2eec61caac2388c71ad8/packages/cicero-cli/lib/commands.js#L618) property is present or not. If present it signs the template before archiving or else it creates the archive without any signatures. The false in previous code for the options argument. So the number of parameters are same for `Commands.archive()` method.